desktop/apps/auction2/client/actions.js

```diff
@@ -71,6 +72,13 @@ export function fetchArtworks() {
         }
       })
 
+      if (filter_sale_artworks.hits && filter_sale_artworks.hits.length > 0) {
```

Comment:
`length` is truthy (0, or greater than 0), so could just do `.length`. (Sometimes `> 0` provides nice metadata for other devs so either way)